benchmarks/mysql-kafka-snapshot/kafka-connect/init/init.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Copyright © 2025 Meroxa, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script install necessary tools and connectors in a Kafka Connect container.
+
+yum install -y jq curl wget
+
+confluent-hub install --no-prompt --verbose debezium/debezium-connector-mysql:latest
```

Comment:
It would be good to use a pinned version of the connector and not `latest`, because whoever is reading the code can more easily find what's the version test. Also, if sometimes in the future we find that KC is slower or faster, we can more easily find the cause for it.

Reply:
Also makes the test repeatable in the future 👍

Reply:
pinned to `2.4.2`